kong/router.lua

```diff
@@ -770,6 +770,27 @@ local function categorize_route_t(route_t, bit_category, categories)
 end
 
 
+local function sanitize_uri_postfix(uri_postfix)
+  if not uri_postfix or uri_postfix == "" then
+    return uri_postfix
+  end
+
+  if uri_postfix == "." or uri_postfix == ".." then
+    return ""
+  end
+
+  if sub(uri_postfix, 1, 2) == "./" then
+    return sub(uri_postfix, 3)
+  end
+
+  if sub(uri_postfix, 1, 3) == "../" then
+    return sub(uri_postfix, 4)
```

Comment:
@dndx,

do you think this should be:

```
    return sub(uri_postfix, 4)
```

or

```
    return sub(uri_postfix, 3)
```

Reply:
See above.